oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1119,7 +1123,8 @@ void pldm::responder::oem_ibm_platform::Handler::setHostEffecterState(
             if (rc != PLDM_SUCCESS)
             {
                 error(
-                    " Set state effecter state command failure. PLDM error code: {RESPONSE_CODE}",
+                    "Set state effecter state command failure for state effecter ID '{EFFECTER_ID}' and instance ID '{INSTANCE}'. PLDM error code: {RESPONSE_CODE}",
+                    "EFFECTER_ID", effecterId, "INSTANCE", instanceId,
```

Comment:
instead of instance id, can we print entityInstance?

Reply:
this is not changed yet?

Reply:
I understood something else and changed that, apologies. Fixed it.